server/api-restful-endpoints/endpoints/accounts.js

```diff
@@ -4,11 +4,11 @@ export function accounts(Api) {
   Api.addCollection(Accounts);
 
   Api.addRoute("accounts", { authRequired: false }, {
-    get: function () {
+    get: () => {
```

Comment:
I was referring to something thing like this 
```js 
{
     get() {},
}

```
But this works too. I will recommend you guys to read on Object destructuring